src/_data/header.yml

```diff
@@ -21,8 +21,8 @@
   icon: blog
   mobileIcon: blog
   blogurl: true
-# - name: Enterprise
-#   icon: enterprise
-#   mobileIcon: enterprise
-#   href: enterprise
-#   landingurl: true
+- name: Enterprise
```

Comment:
![giphy](https://user-images.githubusercontent.com/278641/81455407-252e5180-915d-11ea-94fa-89270b51e8da.gif)

Reply:
You can tell we're serious because there's a little briefcase icon!